setup.py

```diff
@@ -35,10 +38,17 @@
         if ";" in ir:
             entries = ir.split(";")
             extras_require[entries[1]] = entries[0]
-        # Git repos, install master
-        if ir[0:3] == "git":
-            name = ir.split("/")[-1]
-            opt_reqs += ["%s @ %s@main" % (name, ir)]
+        elif ir[0:3] == "git":
```

Comment:
Why do we need that? Did something change?

Reply:
It's to handle this string:
`git+https://github.com/antonlydike/riscemu.git@25d059da090760862f9143478524ed6daf0ab449#egg=riscemu`
The previous format always required main for git strings, but I need a specific commit. The egg also seems required by pip install, and is a better specification of the name than the name of the repository